Sources/Build/BuildPlan.swift

```diff
@@ -1603,9 +1596,10 @@ public class BuildPlan {
             }
         }
 
+        // add test manifest target on non-darwin platforms
         if !buildParameters.triple.isDarwin() {
-            if product.type == .test {
-                linuxMainMap[product].map{ staticTargets.append($0) }
+            if product.type == .test, let testManifestTarget = testManifestTargetsMap[product] {
+                staticTargets.append(testManifestTarget)
```

Comment:
more readable style

Reply:
Agreed.  Some of these Swift expressions are clever but not very approachable, especially for contributors who might be learning Swift.

Reply:
This was introduced in #2918, removing it will break testing the Swift toolchain itself.